flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
no arrch64?

Reply:
Not that I am aware of. There is a mac specific arm64 release, however I have nothing to test this against. I will try to build from source in the future.